docs/installation/installation.md

```diff
@@ -182,14 +183,16 @@ kubectl create secret -n tyk-operator-system generic tyk-operator-conf \
 
 ### Installing cert-manager
 
-If you don't have cert-manager installed, here is a quick install:
+According to [Kubebuilder documentation](https://book.kubebuilder.io/cronjob-tutorial/cert-manager.html#deploying-the-cert-manager),
+cert-manager is the suggested way for provisioning the certificates for the webhook server. If you don't have cert-manager
+installed, a quick installation is as follows:
 
 ```bash
 kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.0/cert-manager.yaml
 ```
 
-Tyk Operator supports Kubernetes v.19+. Ensure cert-manager is compatible with your Kubernetes version.
-the given Kubernetes versions.
+Since Tyk Operator supports Kubernetes v1.19+, minimum cert-manager version is 1.8. Ensure cert-manager is compatible 
```

Comment:
We can skip `Ensure cert-manager is compatible` line, since same thing is already mentioned below.